src/ess/reduce/time_of_flight/types.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+
+from collections.abc import Mapping
+from dataclasses import dataclass
+from typing import NewType
+
+import scipp as sc
+from scippneutron.chopper import DiskChopper
+
+Facility = NewType("Facility", str)
+"""
+Facility where the experiment is performed.
+"""
+
+Choppers = NewType("Choppers", Mapping[str, DiskChopper])
+"""
+Choppers used to define the frame parameters.
+"""
+
+Ltotal = NewType("Ltotal", sc.Variable)
+"""
+Total length of the flight path from the source to the detector.
+"""
+
+SimulationSeed = NewType("SimulationSeed", int)
+"""
+Seed for the random number generator used in the simulation.
+"""
+
+
+NumberOfNeutrons = NewType("NumberOfNeutrons", int)
+"""
+Number of neutrons to use in the simulation.
+"""
+
+
+@dataclass
+class SimulationResults:
+    """
+    Results of a time-of-flight simulation used to create a lookup table.
+    """
+
+    time_of_arrival: sc.Variable
+    speed: sc.Variable
+    wavelength: sc.Variable
+    weight: sc.Variable
+    distance: sc.Variable
```

Comment:
Would be useful to have some docs about the fields here. The generic `Variable` type hint does not say much about, e.g., shape and dims.